lib/parse.js

```diff
@@ -2236,17 +2236,86 @@ function parse($TEXT, options) {
         })
     }
 
+    function import_nameAsterisk() {
+        var start = S.token;
+        var foreign_name;
+        var name;
+
+        next();
+
+        var end = prev();
+
+        name = new AST_Symbol({
```

Comment:
It felt strange to me as well at first,, but I think Import is appropriate here, as this variant of export is first importing the symbol from a foreign file before exporting it.